chapters/object_types.txt

```diff
@@ -136,24 +136,46 @@ float in memory. Implementations are free to always treat the last row
 | bounds   |`FLOAT32_BOX3`| axis-aligned bounding box in world-space (excluding the lights)
 |===================================================================================================
 
-Feature `KHR_TRANSFORMATION_MOTION_BLUR`: Uniformly (in `time`) distributed
-transformation keys can be given with `motion.transform` to achieve
-transformation motion blur (in combination with `time` and `shutter`).
-Alternatively, the transformation keys can also be given as decomposed
-`motion.scale`, `motion.rotation` and `motion.translation`. <<cameras, Camera>>
-`shutter` also needs to be set appropiately.
+Feature: `KHR_INSTANCE_MATRIX`
+
+The `matrix` instance subtype is a basic instance that implements the mandatory parameters and properties listed above.
+
```

Comment:
Should there be explicit language here to say implementations may alias any unsupported instance types to this one to provide a clean fallback?

Reply:
Adressed in 2e53046fabf94b8968b713ed2bd4f2a58d3ddbb0